ts/private/ts_proto_library.bzl

```diff
@@ -21,6 +18,11 @@ def _protoc_action(ctx, proto_info, outputs, options = {
     """
     inputs = depset(proto_info.direct_sources, transitive = [proto_info.transitive_descriptor_sets])
 
+    options = dict(ctx.attr.protoc_gen_options, **{
+        "keep_empty_files": True,
+        "target": "js+dts",
```

Comment:
The way the rule is written, the target NEEDS to be set this way. Please update the rule such that if the user passes any target other than this we tell them it won't work

Reply:
Currently this will overwrite anything the user passes. Should we error instead?